app/src/state/ducks/campaigns.js

```diff
@@ -69,20 +71,43 @@ export const actionCreators = {
 };
 
 // Side Effects, e.g. thunks
-export function createCampaignForGovernment(campaignAttrs) {
+export function createCampaignForGovernment(
```

Comment:
@danmelton Your thoughts on how this was refactored??

Reply:
this will work....but you can attach firstName, lastName and email to the createCampaign api action and it will create both the campaign and the user...

Reply:
That works great, refactoring. Thanks!